DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/src/simple-add-usm.cpp

```diff
@@ -63,19 +63,19 @@ int main() {
   // Create device selector for the device of your interest.
 #if FPGA_EMULATOR
   // DPC++ extension: FPGA emulator selector on systems without FPGA card.
-  ext::intel::fpga_emulator_selector d_selector;
+  const auto& selector = ext::intel::fpga_emulator_selector_v;
 #elif FPGA
   // DPC++ extension: FPGA selector on systems with FPGA card.
```

Comment:
@KanclerzPiotr Do you want to remove the phrase "DPC++ extension" from the comment?

Reply:
I think we should use Intel or oneAPI extension. This is not part of the specification.